txpool/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fuel-txpool"
```

Comment:
we'll need to add these new crates to the tag publish tag validation scripts in CI, and ensure these version match fuel-core and fuel-client.

Reply:
added this two lines to .github/workflows/ci.yml :
```
./.github/workflows/scripts/verify_tag.sh ${{ github.ref_name }} txpool/Cargo.toml
./.github/workflows/scripts/verify_tag.sh ${{ github.ref_name }} interfaces/Cargo.toml
```
and bumbed txpool,interface versions so that they are same as core,client.